config/locales/en/shared.yml

```diff
@@ -64,3 +64,5 @@ en:
       zh: 中文
       zh-hk: 中文
       zh-tw: 中文
+    language_direction:
+      ar: rtl
```

Comment:
There are other languages written right to left such as Hebrew, Persian and Urdu. I don't know if we have any translations in those languages but it would be nice to add support

Reply:
Let's not even get into top-to-bottom writing!

Reply:
Yes, I'm not sure where to get a definitive list from, but I agree. Perhaps these can be added in a later PR when I have found the details

Reply:
Sounds sensible. I found this page: https://www.w3.org/International/questions/qa-scripts.en

Reply:
i've added some more harvested from whitehall and government-frontend